src/modules/auth/authTypes.ts

```diff
@@ -1,18 +1,15 @@
-import type z from 'zod/v4'
-
+import type {
+  GetAuthResponseBody,
+  PostAuthResponseBody,
+  PostAuthResultResponseBody,
+} from '@lokalise/connector-api-contracts'
 import type { ApiReply } from '../commonTypes.ts'
 
-import type { getAuthResponseBody, postAuthResultRequestBody } from './authSchemas.ts'
-
-export type GetAuthResponseBody = z.infer<typeof getAuthResponseBody>
 export type GetAuthResponse = ApiReply<GetAuthResponseBody>
 
 // url field is needed if OAuth flow used
-export type PostAuthResponseBody = Record<string, unknown> & { url?: string }
 export type PostAuthResponse = ApiReply<PostAuthResponseBody>
 
 export type PostAuthRefreshResponse = ApiReply<PostAuthResponseBody>
```

Comment:
probably we can just inline the ApiReply wrapping inside the controllers, but I'd rather do that in a separate PR